pycontrails/datalib/ecmwf/__init__.py

```diff
@@ -6,6 +6,8 @@
 from pycontrails.datalib.ecmwf.era5 import ERA5
 from pycontrails.datalib.ecmwf.hres import HRES
 from pycontrails.datalib.ecmwf.ifs import IFS
+from pycontrails.datalib.ecmwf.model_level_era5 import ModelLevelERA5
```

Comment:
is there a reason to have these outside the `.era5`, `.hres` modules? 

I could see `datalib.ecmwf.era5` containing any ERA5 related datalib, or I do also see the credence in having one class per module, etc.

Not too opinionated here, just want to make sure the org feels logical.

Reply:
I don't have a strong opinion, either--I made this choice mostly for consistency with the ARCO ERA5 datalib, which is in its own file.